README.md

```diff
@@ -139,6 +139,39 @@ only really makes sense if you're calling Grover directly (and not via middlewar
 Grover.new('<some URI with basic authentication', username: 'the username', password: 'super secret').to_pdf
 ```
 
+#### Adding cookies
+If you need to set cookies to your new page instance, just pass them along like this:
+
+```ruby
+myCookies = [{:name=>"sign_username", :value=>"any@any.com", :domain=>"mydomain"}, {:name=>"_session_id", :value=>"9c014df0b699d8dc08d1c472f8cc594c", :domain=>"mydomain"}]
+Grover.new('<some URI with cookies', cookies: myCookies ).to_pdf
+```
+
+If, let's say, you wanted to just forward the cookies that you got on your original request, you can do something like:
+
+```ruby
+def header_cookies(req=nil)
```

Comment:
the spacing in your example looks a bit messed up. Could you please clean it up?